libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -108,24 +108,24 @@ const AP_Param::GroupInfo RC_Channel::var_info[] = {
     // @DisplayName: RC input option
     // @Description: Function assigned to this RC channel
     // @Values{Copter, Rover, Plane, Blimp}: 0:Do Nothing
-    // @Values{Copter}: 2:Flip
+    // @Values{Copter}: 2:Mode FLIP
     // @Values{Copter}: 3:Simple Mode
     // @Values{Copter, Rover}: 4:RTL
-    // @Values{Plane}: 4:ModeRTL
+    // @Values{Plane}: 4:Mode RTL
```

Comment:
If there the same for vehicles you can combine them.

`// @Values{Copter, Rover, Plane}: 4:Mode RTL`

Reply:
will do....